torontoadvotech/client
# Ws 11

## Description



- Build out Partners page with form submission to trigger modal based on [design](https://www.figma.com/file/AGjAHUe7SnUJPg7hxKzRbd/TOWiT-File)

## Related Issue


[WS-11](https://torontoadvotech.atlassian.net/browse/WS-11)

## Acceptance Criteria



Desktop screens can be found in the figma file. Unfortunately we don’t have mobile screens for this page so just use your best judgement, it doesn’t need to be perfect.

To create the form follow the same structure as in the sign up / login / update profile form. 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### After



![Screenshot_2021-01-25 WorkSMART Mentorship](https://user-images.githubusercontent.com/49457972/105781220-2cda9400-5f40-11eb-93a4-419839ffb577.png)


## Testing Steps / QA Criteria


1. Check out to `WS-11` and run local
2. Fill out and submit form. On form submit, modal to confirm submission will show
